src/game/boe.main.cpp

```diff
@@ -602,13 +602,14 @@ static void replay_next_action() {
 		return;
 	}else if(t == "handle_toggle_active"){
 		handle_toggle_active(need_reprint);
-		return;
 	}else if(t == "handle_parry"){
 		handle_parry(did_something, need_redraw, need_reprint);
 	}else if(t == "handle_monster_info_menu"){
 		handle_monster_info_menu(boost::lexical_cast<int>(next_action.GetText()));
+		return;
 	}else if(t == "cancel_item_target"){
 		cancel_item_target();
+		return;
```

Comment:
This seems wrong (though in this case it's that I probably should've _made_ it advance time and didn't). Assuming time advances when casting Identify outdoors, it should also advance when calling Identify in town. (The item target mode is only used in town.)